frontend/src/components/log-file/index.vue

```diff
@@ -287,20 +294,10 @@ onMounted(async () => {
 });
 
 onUnmounted(() => {
+    console.log('onUnmounted');
     onCloseLog();
 });
 
-onMounted(async () => {
-    firstLoading.value = true;
-    await init();
-    nextTick(() => {
-        if (logContainer.value) {
-            logContainer.value.scrollTop = totalHeight.value;
-            containerHeight.value = logContainer.value.getBoundingClientRect().height;
-        }
-    });
-});
-
 defineExpose({ changeTail, onDownload, clearLog });
 </script>
 <style lang="scss" scoped>
```

Comment:
The code looks fine and there is no evidence of an issue, irregularity or optimization. It seems that this project uses Vue.js to generate log views on mobile devices which could be accessed from the web, using a custom script with Vue router's functionality.